toolchain/check/handle_struct.cpp

```diff
@@ -125,8 +125,8 @@ auto HandleParseNode(Context& context, Parse::StructTypeLiteralId node_id)
   context.node_stack()
       .PopAndDiscardSoloNodeId<Parse::NodeKind::StructTypeLiteralStart>();
 
-  CARBON_CHECK(refs_id != SemIR::InstBlockId::Empty)
-      << "{} is handled by StructLiteral.";
+  CARBON_CHECK(refs_id != SemIR::InstBlockId::Empty,
+               "{} is handled by StructLiteral.");
```

Comment:
Should this be escaped for formatv?

Reply:
Weirdly, this works as is... because we avoid `formatv` when there are no arguments.

But your comment is worrisome, because that seems like an unfortunate implementation detail to leak in this way. I'm not really sure what the best approach is here, and interested in ideas.

Reply:
Can it just pass to formatv irrespective?

Reply:
Note, if you're concerned about codesize of that, the version with no arguments could do just a thin non-templated wrapper of formatv. Just to force that the argument string gets the same treatment.

But it's probably still going to be missed, unless we get something with compile-time validation.

Reply:
So, it actually *is* passed to `formatv` always, I misremembered. So I think we're fine here.